modules/apps/site-initializer/site-initializer-raylife-d2c/site-initializer-raylife-d2c-test/src/testFunctional/tests/RaylifeSite.testcase

```diff
@@ -3,7 +3,7 @@ definition {
 
 	property osgi.modules.includes = "site-initializer-raylife-d2c";
 	property portal.release = "false";
-	property portal.upstream = "true";
+	property portal.upstream = "talitha";
```

Comment:
Hi @itstali 
This value of portal.upstream can't be merged. Would you please change back to "true"?